components/posts/ContentRenderer.tsx

```
@@ -0,0 +1,586 @@
+          border: "border-yellow-500/30",
+          text: "text-yellow-300",
+          icon: "⚠️",
+        },
+        danger: {
+          bg: "bg-red-500/10",
+          border: "border-red-500/30",
+          text: "text-red-300",
+          icon: "🚨",
+        },
+      };
+      const variant = variants[item.variant] || variants.info;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)